### PR TITLE
add annotations [run_process_replay]

### PR DESCRIPTION
**Pull request by DKormann**

### CODE REVIEW

File: tinygrad/codegen/uops.py [hunk 1]
```diff
@@ -289,7 +289,7 @@ def print(self):
     for i,u in enumerate(self):
       print(f"{i:4d} {str(u.op):20s}: {str(u.dtype) if u.dtype is not None else '':25s} " f"{str([self.uops.index(x) for x in u.src]):32s} {u.arg}")
 
-  def graph_rewrite(self, sink, pm):
+  def graph_rewrite(self, sink: UOp, pm: PatternMatcher):
```
Comment (chenyuxyz) [inline tinygrad/codegen/uops.py:292]: we don't have a strict style guide, but i'd remove the spaces in between like other functions in this file

Comment (geohot) [reply tinygrad/codegen/uops.py:292]: Is there a way to have the linter check this?

Comment (DKormann) [reply tinygrad/codegen/uops.py:292]: ok

Comment (DKormann) [reply tinygrad/codegen/uops.py]: is this  a rule for only type annotations or also eg. lambda function

**Comment by geohot** [comment]
I don't think we should change any until there's a linter rule.

**Pull request closed without merging**